docs/eventing/experimental-features.md

```diff
@@ -85,3 +85,36 @@ With this feature you can allow Knative to resolve the full `APIVersion` and fur
 
 !!! note
     At the moment this feature is implemented only for `Subscription.Spec.Subscriber.Ref`.
+
+### `DeliverySpec.Timeout` field
```

Comment:
```suggestion
### DeliverySpec.Timeout field
```
I think we're not using markup in headings in our style guide

Reply:
Ok then we need to change the KReference.Group header as well